source/_lovelace/sidebar.markdown

```diff
@@ -12,7 +12,7 @@ This view doesn't have support for badges.
 To change a view to edit mode, or to change the location of a card, enable edit mode:
 Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**.
 
-You can set if a card should be placed in the main (left) column of the sidebar column (right), by pressing the arrow right or left arrow in the bar underneath the card.
+You can set if a card should be placed in the main (left) column or in another column (right) of the sidebar, by pressing the right or left arrow in the bar underneath the card.
```

Comment:
It now suggests there is a third column right of the sidebar, which is incorrect.